examples/quickstart/jupyter-notebooks/druidapi/README.md

```diff
@@ -36,21 +39,27 @@ At present, the best way to use `druidapi` is to clone the Druid repo itself:
 git clone git@github.com:apache/druid.git
 ```
 
-`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`
+`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
+From this directory, install the package and its dependencies with pip using the following command:
+
+```
+pip install .
+```
 
-Eventually we would like to create a Python package that can be installed with `pip`. Contributions
-in that area are welcome.
+Note that there is a second level `druidapi` directory that contains the modules. Do not run
+the install command in this internal folder.
```

Comment:
We usually refer to it as a child or subdirectory and not an internal folder, right?

Reply:
```suggestion
the install command in the subdirectory.
```